src/shared/components/post/post-listing.tsx

```diff
@@ -214,7 +214,15 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     if (url && isVideo(url)) {
       return (
         <div className="embed-responsive ratio ratio-16x9 mt-3">
-          <video muted controls className="embed-responsive-item col-12">
+          <video
+            onLoadStart={this.handleVideoLoadStart}
```

Comment:
Please use `linkEvent` for these as described in [inferno's docs](https://www.infernojs.org/docs/guides/event-handling).